src/highdicom/base.py

```diff
@@ -127,10 +127,7 @@ def __init__(
 
         # Include all File Meta Information required for writing SOP instance
         # to a file in PS3.10 format.
-        self.preamble = b'\x00' * 128
         self.file_meta = FileMetaDataset()
```

Comment:
```suggestion
        self.preamble = b'\x00' * 128
        self.file_meta = FileMetaDataset()
```

Reply:
Turns out I was wrong about this. I reintroduced the attribute via ef8f0756ef858025b57d5c15814274eb6155184b